ansible/roles/ooni-collector/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+# FIXME: do we _really_ need nginx in front of the collector?
+- template: src=ngx-ooni-backend-web dest=/etc/nginx/sites-enabled/ooni-backend-web
+  notify: reload nginx
+
+# FIXME what is this? Do we need this for the collector?!
+- name: docker network for backend
+  docker_network:
+    name: msm
+    driver_options:
+      com.docker.network.bridge.name: brmsm
+    ipam_options:
+      subnet: 172.25.232.0/24
+      gateway: 172.25.232.1
+
+# FIXME this is not the correct procedure to start it up
+# at the moment this is copied from ooni-measurements
+- name: ooni-backend webservice
+  docker_container:
+    image: openobservatory/ooni-measurements:{{ ooni_backend_tag }}
```

Comment:
so this is actually not the correct docker container in here, right?

Reply:
Yeah, this is copied and pasted from the other file you gave me and needs updating.

Reply:
(So, now we use docker images for services? I guess this would make things simpler a great deal, but it was not the case when I looked into proteus deployment, so I didn't understood this bit.)